Packages/com.unity.inputsystem/Documentation~/Gamepad.md

```diff
@@ -198,3 +198,37 @@ The Input System support Switch Pro controllers on desktop computers via the [`S
 ## Cursor Control
 
 To give gamepads and joysticks control over a hardware or software cursor, you can use the [`VirtualMouseInput`](../api/UnityEngine.InputSystem.UI.VirtualMouseInput.html) component. See [`VirtualMouseInput` component](UISupport.md#virtual-mouse-cursor-control) in the UI section of the manual.
+
+## Discover all connected devices
+
+There are various ways to discover the currently connected devices, as shown in the code samples below.
+
+To query a list of all connected devices (does not allocate; read-only access):
```

Comment:
This isn't needed for the add/remove onDeviceChange part so could go down with the "To find all gamepads and joysticks" section